addons/utm/models/utm.py

```diff
@@ -42,14 +49,84 @@ def _group_expand_stage_ids(self, stages, domain, order):
         stage_ids = stages._search([], order=order, access_rights_uid=SUPERUSER_ID)
         return stages.browse(stage_ids)
 
+    def _merge_utm_campaigns(self, campaign=False, name=False, user_id=False, stage_id=False, tag_ids=False):
+        if len(self.ids) <= 1:
+            raise UserError('Please select more than one campaign from the list.')
+        if not all(r.active for r in self):
+            raise UserError('Only active campaigns can be merged.')
+        if campaign and campaign not in self:
+            raise UserError('Kept campaign should be one of the merged campaign.')
```

Comment:
Something I missed during the first review:

You need to translate all the strings that are shown to the user.
Translation is done by simply prefixing with the "_" function.

So here it would be for example: `raise UserError(_("Please select more than one campaign from the list."))`
(You also need to `from odoo import _` of course).

When you do that, if the user has set his language to french, the sentence will be displayed in french.